source/module_hamilt_pw/hamilt_pwdft/initcontroller.cpp

```diff
@@ -0,0 +1,256 @@
+#include "initcontroller.h"
```

Comment:
how about change the file name to psi_init ?

Reply:
In fact, we have class named psi_initializer and Initcontroller class has a member of "psi_initializer* psi_init". So it is not good to change the name to psi_init. However, the name "initcontroller" lacks the information of psi. It is not good, either.

Reply:
How about wfinit_ctrl?